Frends.CSV.Parse/Frends.CSV.Parse.UnitTests/UnitTests.cs

```diff
@@ -328,4 +328,35 @@ public void TestParseTreatMissingFieldsAsNullDefaultValue()
 

```

Comment:
https://github.com/FrendsPlatform/Frends.CSV2/blob/58ab28316ff6947de621c080b23cc52318dfc53d/Frends.CSV.Parse/Frends.CSV.Parse.UnitTests/UnitTests.cs#L321
You can omit the return, no need to save to variable which is not used.